Tasks/Delay/task.json

```diff
@@ -0,0 +1,34 @@
+{
+    "id": "28782B92-5E8E-4458-9751-A71CD1492BAE",
+    "name": "Delay",
```

Comment:
As discussed, not adding Preview flag as task is not being ported to M122 snapshot.